scripts/casefold_db.py

```diff
@@ -262,32 +258,27 @@ def update_global_assoc(sydent, db: sqlite3.Connection, flag):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Casefold email addresses in database")
     parser.add_argument(
-        "--no_email", action="store_true", help="run script but do not send emails"
+        "--no-email", action="store_true", help="run script but do not send emails"
     )
-    parser.add_argument("--apply", action="store_true", help="run full script")
     parser.add_argument(
-        "--dry_run",
+        "--dry-run",
         action="store_true",
         help="run script but do not send emails or alter database",
     )
 
-    args = vars(parser.parse_args())
+    parser.add_argument("config_path", help="path to the sydent configuration file")
 
-    count = 0
-    for _, value in args.items():
-        if value:
-            count += 1
-            flag = value
+    args = parser.parse_args()
 
-    # exit if we have been given more than one flag or no flag
-    if count > 1:
-        sys.exit("Please specify only one flag to execute, refer to -help for options.")
-    elif count == 0:
-        sys.exit("Please specify a flag to execute, refer to -help for more help.")
+    # if the path the user gives us doesn't work, find it for them
+    if os.path.exists(args.config_path):
+        config = parse_config_file(args.config_path)
+    else:
+        config_file_path = os.environ.get("SYDENT_CONF", "sydent.conf")
+        config = parse_config_file(config_file_path)
```

Comment:
I think this will possibly be confusing when someone make a typo and it doesn't do what they expect.
Instead, it's probably kinder to give an error and stop.

```suggestion
    if not os.path.exists(args.config_path):
        print(f"The config file '{args.config_path}' does not exist.")
        os.exit(1)

     config = parse_config_file(config_file_path)
```